### DISCUSSION TITLE
Is it possible to use Open Shell on Linux, on the Ubuntu linux distribution?

### DISCUSSION THREAD

**Discussion by ghost** [Open-Shell/Open-Shell-Menu#1126] [Q&A] [answered]
Hi. I would like to modify my graphical environment the way I want. Is it possible to use Open Shell on Linux, on the Ubuntu linux distribution?

**Comment by blackcrack** [comment, marked as answer]
you can use KDE/Plasma there be different  Startmenu /kicker possible.
My self have make different Icon and color themes for KDE/Plasma, you can find it, 
if you let search for "blacky" in Plasma on/at the Systemsettings5
( i use Win7 as workstation and Linux in the basement as Server on my net-domain)
I be a ex Mandrake/Mandriva User/Member and use Linux as client by time  to time..
but the whole troubles at OpenMandriva, had i change to Fedora.. because Mandriva 
is no more this Mandrake as it was.. i be total disappeared behind the whole years ..
killing the draktools, killing the urpmi ... the head things of Mandrake is gone and
because this, it's better to use Fedora before i touch the crapdistro again .. :\\
Because that can i say because i know ;) So, use KDE/Plasma and you have a nice Windowmanager as client-Surface..

best

**Comment by TheJaredWilcurt** [comment]
For Gnome desktops (Ubuntu, Zorin, etc) there is a Gnome extension called "ArcMenu"

* https://extensions.gnome.org/extension/3628/arcmenu/

It is dramatically better than anything else on Linux that I've found. However, it is also dramatically worse than Open-Shell. Welcome to Linux, everything sucks and the people that can change it don't care because they still think humans should be spending more than 30 seconds a day in the command line.
